lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -204,6 +209,11 @@ Function Choco-Upgrade
         $cmd += " -ignoredependencies"
```

Comment:
They all exist and they all work AFAIK. Often chocolatey also has 4 options for the same thing. Bad design IMO I guess they took the "be liberal to what you accept" very serious here.